scripts/thonk.coffee

```diff
@@ -0,0 +1,28 @@
+# Description:
+#   Thonk is love. Thonk is life
+#
+# Dependencies:
+#   None
+#
+# Configuration:
+#   None
+#
+# Commands:
+#   hubot thonk me - Receive a thonk
+#   hubot thonk bomb N - get N thonks
+
+module.exports = (robot) ->
+  thonkArray = [
+    'https://i.redd.it/7geba60hr0e01.jpg',
+    'http://i0.kym-cdn.com/photos/images/original/001/279/809/172.gif',
+    'https://i.imgur.com/zXAA3CV.gif',
+    'https://i.imgur.com/hlP0Brh.gif',
+    'https://i.imgur.com/8cLO9So.gif'
+  ]
```

Comment:
This doesn't seem like enough thonks IMHO

Reply:
:man_shrugging: there are only 6 pugs so I figured 5 thonks was ok

Reply:
Ok I added 3 more thonks